site/src/component/GradeDist/Chart.tsx

```diff
@@ -70,61 +70,46 @@ export default class Chart extends React.Component<ChartProps> {
         id: 'A',
         label: 'A',
         A: gradeACount,
-        color: '#2484C6',
```

Comment:
Missing `colors[0]`

Although ideally, we use a typescript enum to name the colors instead of just putting them in a list.

```ts
export enum GradeColors {
  A = '#2484C6',
  B = '#54B058',
  // ...
}
// or Blue = '#2484C6', Green = '#54B058', ...
```

```ts
color: GradeColors.A // or color: GradeColors.Blue
```